src/pke/include/key/keypair.h

```diff
@@ -52,9 +52,13 @@ class KeyPair {
     explicit KeyPair(PublicKeyImpl<Element>* a = nullptr, PrivateKeyImpl<Element>* b = nullptr)
         : publicKey(a), secretKey(b) {}
 
-    bool good() {
+    bool is_allocated() const {
```

Comment:
as good() is already called in the code, i would make the existing function good() "const" and have is_allocated() call good().

Reply:
Sure, no problem!

Reply:
I will change this commit as you suggested.